### DISCUSSION TITLE
SMG Mk. III 'Interceptor' (IMPERIAL) [C]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#53] [Card Balance]
SMG Mk. III 'Interceptor' (IMPERIAL) [C]
EQUIPMENT, FIREARM
[S] +2. Opponent [S] -1

My suggestion :
EQUIPMENT, FIREARM, NOACTION
[S] +2. Opponent [S] -1

**Comment by Michalx11** [comment]
I like this idea

**Comment by Kite80** [comment]
ç

Il Mar 8 Giu 2021, 04:44 Michalx11 ***@***.***> ha scritto:

**Comment by Sol-au-fein** [comment]
Good idea. It is basicaly better Punisher Handgun exclusively for Imperial.
